app/controllers/api/account.php

```diff
@@ -1498,6 +1498,12 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
                         'providerType' => MESSAGE_TYPE_EMAIL,
                         'identifier' => $email,
                     ]));
+
+                    $queueForEvents
```

Comment:
Just thinking out loud, will we have same problem in Magic URL, email OTP, or SMS logins? When they create new user, is the event triggered properly?

Reply:
We can apply the same fix across them, or perhaps rethink events a little bit. In general we have a problem where many endpoints can now trigger multiple events depending on the scenerio